src/components/SectionList/index.js

```diff
@@ -0,0 +1,3 @@
+import {SectionList} from 'react-native';
+
+export default SectionList;
```

Comment:
What does this file do ?

Reply:
Not much 😄 . It imports the unmodified `SectionList` and then re-exports it. This is because we have different builds depending on the platform and will switch extensions e.g.

```js
import SectionList from './SectionList';
```
On web, this will pull from `/SectionList/index.js` but on Android it will use `/SectionList/index.android.js`

So we are basically saying, when building for web use the unmodified `SectionList`